ui/pages/send/send-content/send-content.component.js

```diff
@@ -83,24 +91,53 @@ export default class SendContent extends Component {
     );
   }
 
+  checkContractAddress = async () => {
+    const { recipient, tokenAddressList } = this.props;
+
+    const symbolAndDecimals = await getSymbolAndDecimals(
```

Comment:
Once controllers v25.0.0 is added to the extension we'll have a way check whether the address is an ERC721 or ERC1155 contract as well. Should we wait and add that functionality? I'm happy to pair program and make that change as well. cc @danjm @david0xd

Reply:
I think that can be a separate PR after this PR merges. We should create a separate github ticket for that:

- [ ] Create github ticket for ensuring the known contract address warning in the send flow also apples to NFT tokens